apps/vc-api/src/vc-api/exchanges/exchange.service.spec.ts

```diff
@@ -60,13 +60,17 @@ describe('ExchangeService', () => {
       transaction = entity;
     })
   }));
-  let exchange: ExchangeEntity;
-  const repositoryMockFactory = jest.fn(() => ({
-    findOne: jest.fn(() => exchange),
-    save: jest.fn((entity) => {
-      exchange = entity;
-    })
-  }));
+
+  const repositoryMockFactory = jest.fn(() => {
+    let exchange: ExchangeEntity;
+
+    return {
+      findOne: jest.fn(() => exchange),
+      save: jest.fn((entity: ExchangeEntity) => {
+        exchange = entity;
+      })
```

Comment:
I know it's outside of the task scope, but i think `save` should actually return saved value, to met `typeorm` behavior